Yank/reports/notebook.py

```diff
@@ -345,43 +346,57 @@ def generate_mixing_plot(self, mixing_cutoff=0.05, mixing_warning_threshold=0.90
             cmap = plt.get_cmap("Blues")
         else:
             cmap = LinearSegmentedColormap('BlueWarnRed', cdict)
+
+        # Plot a diffusing mixing map for each phase.
         for phase_name, subplot in zip(self.phase_names, subplots):
+            # Generate mixing statistics.
             analyzer = self.analyzers[phase_name]
-            mixing_data, mu = analyzer.generate_mixing_statistics(number_equilibrated=self.nequils[phase_name])
-            # Without vmin/vmax, the image normalizes the values to mixing_data.max which screws up the warning colormap
-            # Can also use norm=NoNorm(), but that makes the colorbar manipulation fail
-            output_image = subplot.imshow(mixing_data, aspect='equal', cmap=cmap, vmin=0, vmax=1)
-            # Add colorbr
+            mixing_statistics = analyzer.generate_mixing_statistics(
+                number_equilibrated=self.nequils[phase_name])
+            transition_matrix, eigenvalues, statistical_inefficiency = mixing_statistics
+
+            # Without vmin/vmax, the image normalizes the values to mixing_data.max
+            # which screws up the warning colormap.
+            # Can also use norm=NoNorm(), but that makes the colorbar manipulation fail.
+            output_image = subplot.imshow(mixing_statistics.transition_matrix, aspect='equal',
```

Comment:
Why not just use `transition_matrix` here since you split it off in the line above?

Reply:
Thanks! I didn't notice it. Will do.